### PR TITLE
Fix account preset link in extensibility.adoc

### PR DESCRIPTION
**Pull request by ivpavici**
chore: changed broken link for Account contract preset

Fixes #???? 

#### PR Checklist

- [ ] Tests
- [x] Documentation
- [ ] Changelog entry

### CODE REVIEW

File: docs/modules/ROOT/pages/extensibility.adoc [hunk 1]
```diff
@@ -79,7 +79,7 @@ They can be deployed as-is or used as templates for customization.
 
 Some presets are:
 
-* https://github.com/OpenZeppelin/cairo-contracts/blob/ad399728e6fcd5956a4ed347fb5e8ee731d37ec4/src/openzeppelin/account/Account.cairo[Account]
+* https://github.com/OpenZeppelin/cairo-contracts/blob/main/src/openzeppelin/account/presets/Account.cairo[Account]
```
Comment (martriay) [inline docs/modules/ROOT/pages/extensibility.adoc:82]: we avoid linking to `main` directly to preven broken links down the road. We prefer permalinks, this is, linking to a specific commit hash.

**Review by martriay — CHANGES_REQUESTED** [review]
Thanks for doing this! Left a small comment.

File: docs/modules/ROOT/pages/extensibility.adoc (hunk 1 above)
Comment (ivpavici) [reply docs/modules/ROOT/pages/extensibility.adoc]: thanks! In that case, I've also changed the link below to permalink :)

Comment (martriay) [reply docs/modules/ROOT/pages/extensibility.adoc]: Good catch!

**Review by martriay — APPROVED** [review]

**Pull request merged**